src/Angor/Client/Pages/Investor.razor

```diff
@@ -431,6 +482,10 @@
                                 {
                                     projects.Add(investorProject);
                                     storage.AddInvestmentProject(investorProject);
+                                    
+                                    // todo: David to check this, not sure this is correct.
+                                    // storage.SetNostrPublicKeyPerProject(investorProject.ProjectInfo.ProjectIdentifier, investorNostrPrivateKey.PubKey.ToHex()[2..]);
```

Comment:
@DavidGershony I believe this is correct and can be commented back, can you double check?